src/electionguard/serializable.py

```diff
@@ -86,14 +87,54 @@ def write_json_file(json_data: str, file_name: str, file_path: str = "") -> None
         json_file.write(json_data)
 
 
+ENCODE_THRESHOLD: Final[int] = 100_000_000
+
+
+def int_to_maybe_base64(i: int) -> Union[str, int]:
+    """
+    Given a non-negative integer, returns a big-endian base64 encoding of the integer,
+    if it's bigger than `ENCODE_THRESHOLD`, otherwise the input integer is returned.
+    :param i: any non-negative integer
+    :return: a string in base64 or just the input integer, if it's "small".
+    """
+    assert i >= 0, "int_to_maybe_base64 does not accept negative numbers"
```

Comment:
We do not want `assert` being hit in production functions outside of tests. This likely should just throw an out of range error. This is the correct behavior.

Reply:
Can do.